Makefile

```diff
@@ -110,9 +110,21 @@ endif
 
 all: lint test manager
 
+## Location to install custom assets
+CUSTOMENVTEST = $(LOCALBIN)/k8s/1.29.1-linux-amd64
+$(CUSTOMENVTEST):
+	if [ ! -d "$(CUSTOMENVTEST)" ]; then \
+		mkdir -p $(LOCALBIN)/k8s/1.29.1-linux-amd64; \
+		curl -L https://sertaccdn.azureedge.net/kube-vap-fix/etcd --output $(LOCALBIN)/k8s/1.29.1-linux-amd64/etcd && chmod +x $(LOCALBIN)/k8s/1.29.1-linux-amd64/etcd; \
```

Comment:
this is a custom 1.28 build from https://github.com/kubernetes/kubernetes/pull/123477

Reply:
we can build for 1.29 if that's preferred